lms/templates/fields/field_readonly.underscore

```diff
@@ -1,10 +1,8 @@
-<label class="u-field-title" aria-hidden="true">
-    <%- title %>
-</label>
-<span class="sr" for="u-field-input-<%- id %>"><%- screenReaderTitle %></span>
-<span class="u-field-value">
-    <input id="u-field-input-<%- id %>" aria-describedby="u-field-message-<%- id %>" type="text" name="input" readonly=true value="<%- value %>">
-</span>
+<% if (title) { %>
+<span class="u-field-title" aria-hidden="true"><%- title %></span>
```

Comment:
When instantiating the component, a screenReaderTitle can be specified as an option independent of the title. When rendering the template, this is passed in: screenReaderTitle = this.options.screenReaderTitle || this.options.title.

Example differences (there are more than this)
screenReaderTitle = "Reset your Password" vs. "Password" (accounts page)
screenReaderTitle = "Username" vs. nothing (profile page)
screenReaderTitle = "Country" vs. nothing (profile page)
screenReaderTitle = "Preferred Language" vs. nothing (profile page)